### DISCUSSION TITLE
Preserve UV sets

### DISCUSSION THREAD

**Discussion by sergin3d2d** [zeux/meshoptimizer#552] [Q&A]
Hello! Is there a way to preserve multiple UV sets on geometry during optimization? Currently, I have to add extra textures that linked to another UV set and it helps to save it, but this doesn't always work smoothly during export.

**Comment by zeux** [comment]
The only way is to have textures that reference them. This is both due to the fact that without textures, there's no hint as to whether the UV sets are needed, but also because for quantization, without texture references it's impossible to preserve the values as it requires putting the quantization parameters into KHR_texture_transform extension, which is not available without, well, textures.
